phpunit.xml.dist

```diff
@@ -20,12 +39,11 @@
 			<directory>tests</directory>
 		</testsuite>
 	</testsuites>
-
-	<coverage/>
-
 	<source>
 		<include>
-			<directory>src/Packetery</directory>
+			<directory>src</directory>
 		</include>
 	</source>
+
+	<coverage includeUncoveredFiles="false"/>
```

Comment:
`includeUncoveredFiles` bych nechal, nevidím důvod nepřiznávat neotestované části, je to technický dluh, ne záměr.